wordpress-mobile/gutenberg-mobile
# Fix notice size overflow

`gutenberg` PR: https://github.com/WordPress/gutenberg/pull/31083

**To test:**

For testing this, manual code changes are required to test different text sizes. I tested this by modifying the text of the notice that is triggered when copying a block:

1. Go to this line
1. Change the text (it would be nice to test with different text sizes like 2/3/4 lines)
1. Select any block
1. Tap on the three dots button to open the block actions menu
1. Tap on "Copy block" (this will display a notice)
1. Observe that the content of the notice is displayed properly

PR submission checklist:
